### PR TITLE
GetCapabilities and GetBoundingBox methods

### PR DESCRIPTION
**Pull request by almostcake**

### CODE REVIEW

File: src/WFS.js [hunk 1]
```diff
@@ -163,14 +169,127 @@ L.WFS = L.FeatureGroup.extend({
...
+        if (exceptionReport) {
+          if (typeof (errorCallback) === 'function') {
+            errorCallback(exceptionReport.message);
+          }
+
+          return;
+        }
+
+        // Request was truly successful (without exception report), parse WFS_Capabilities.
+        capabilities = L.XmlUtil.parseXml(data).documentElement;
```
Comment (SeregaMatin) [inline src/WFS.js]: Parsing can potentially be failed, it must be handled and then 'errorCallback' must be called.

File: src/WFS.js [hunk 2]
```diff
@@ -163,14 +169,127 @@ L.WFS = L.FeatureGroup.extend({
...
+      },
+      error: function (errorMessage) {
+        if (typeof (errorCallback) === 'function') {
+          errorCallback(errorMessage);
+        }
+      }
+    });
+  },
+
+  getBoundingBox: function (successCallback, errorCallback) {
```
Comment (SeregaMatin) [inline src/WFS.js:248]: Unit tests for 'getBoundingBox' are necessary. To save time you could get them from Leaflet-WMS plugin and adapt them to this plugin.

File: src/WFS.js [hunk 3]
```diff
@@ -163,14 +169,127 @@ L.WFS = L.FeatureGroup.extend({
...
         that.fire('error', {
           error: new Error(errorMessage)
         });
 
         return that;
       }
     });
+  },
+
+  getCapabilities: function (successCallback, errorCallback) {
```
Comment (SeregaMatin) [inline src/WFS.js]: Unit tests for 'getCapabilities' are necessary. To save time you could get them from Leaflet-WMS plugin and adapt them to this plugin.

File: src/WFS.js (hunk 1 above)
Comment (SeregaMatin) [reply src/WFS.js]: Parsing errors are still unhandled.
Also there are some conflicts with 'develop' brunch.

File: spec/WFS.js [hunk 4]
```diff
@@ -386,6 +421,10 @@ describe('WFS', function () {
       server.respond();
 
       // Check events handlers.
+      expect(onLoadEventHandler.calledWithMatch({
+        responseText: getFeatureResponseText
+      })).to.be.equal(true);
+      expect(onErrorEventHandler.notCalled).to.be.equal(true);
```
Comment (kuzkok) [inline spec/WFS.js]: wrong merge, same expect 2 lines below

File: spec/WFS.js [hunk 5]
```diff
@@ -106,7 +137,6 @@ describe('WFS', function () {
         maxFeatures: 5000
       };
 
-      sinon.stub(L.WFS, 'describeFeatureType');
```
Comment (kuzkok) [inline spec/WFS.js]: why you remove it?

File: spec/WFS.js [hunk 6]
```diff
@@ -91,7 +122,7 @@ describe('WFS', function () {
   describe('#getFeature', function () {
     var feature;
 
-    before(function() {
+    beforeEach(function () {
```
Comment (kuzkok) [inline spec/WFS.js]: for which purposes you need here berforeEach?

File: spec/WFS.js (hunk 4 above)
Comment (almostcake) [reply spec/WFS.js]: others are too merge mistakes, fixed

**Pull request merged**